multinet/multinet.py

```diff
@@ -53,6 +64,7 @@ def _graphql():
     app.logger.debug("variables: %s" % variables)
 
     result = graphql_query(query, variables)
+
```

Comment:
What was the reason for this change?

Reply:
There's no reason for this extra line. Removed in the next commit